concent_api/gatekeeper/views.py

```diff
@@ -35,9 +36,13 @@
 @require_POST
 def upload(request):
     logger.debug("Upload request received.")
+
     if request.content_type in ['multipart/form-data', '', None] or request.content_type.isspace():
         logger.info(f'Unsupported content type: {request.content_type}')
-        return gatekeeper_access_denied_response('Unsupported content type.')
+        return gatekeeper_access_denied_response(
+            'Unsupported content type.',
+            ErrorCode.HEADER_CONTENT_TYPE_NOT_SUPPORTED.value,
```

Comment:
Let's use enum items, not names or values. By taking the value immediately we lose type information and only make expressions longer. Enum items should be converted to values only when they need to be serialized, i.e. in  `gatekeeper_access_denied_response()`.

Reply:
:+1: